pkg/exchange/coinbase/Makefile

```diff
@@ -0,0 +1,7 @@
+generate: symbols go-generate
+
+symbols:
+	@echo "Generate symbols.go" && go run ./generate_symbol_map.go
+
+go-generate:
+	@echo "Running \`go generate\`" && go generate ./...
```

Comment:
you don't need the makefile actually? you can embed them in the go:generate comment

Reply:
I just want a way to document how I generate the `symbols.go`